### ISSUE TITLE
[1.2.0 beta1] tar ball contains backup files in locales/po

### ISSUE THREAD

**Issue by paulgevers** [Cacti/cacti#2071]
It seems the tar ball contains multiple files that aren't supposed to be there.

```
paul@testavoira ~/cacti $ ls -1 locales/po/*~
locales/po/bulgarian_bulgaria.po~
locales/po/chinese_china_simplified.po~
locales/po/dutch_netherlands.po~
locales/po/french_france.po~
locales/po/german_germany.po~
locales/po/japanese_japan.po~
locales/po/russian_russia.po~
locales/po/spanish_spain.po~
locales/po/swedish_sweden.po~
```

**Comment by netniV** [comment]
That would appear to be correct, the ~ are usually backup files.

**Comment by ronytomen** [comment]
Fixed in commit f8f40fce9db2460314d4355138c9d2d536b26523
